commands/shutdown.md

```diff
@@ -1,35 +1,40 @@
 The command behavior is the following:
 
+* If there are any replicas lagging behind in replication:
+  * Pause clients attempting to write by performing a `CLIENT PAUSE` with the `WRITE` option.
+  * Wait up to the configured `shutdown-timeout` (default 10 seconds) for replicas to catch up the replication offset.
 * Stop all the clients.
 * Perform a blocking SAVE if at least one **save point** is configured.
 * Flush the Append Only File if AOF is enabled.
 * Quit the server.
 
 If persistence is enabled this commands makes sure that Redis is switched off
 without the lost of any data.
-This is not guaranteed if the client uses simply `SAVE` and then `QUIT` because
-other clients may alter the DB data between the two commands.
 
 Note: A Redis instance that is configured for not persisting on disk (no AOF
 configured, nor "save" directive) will not dump the RDB file on `SHUTDOWN`, as
 usually you don't want Redis instances used only for caching to block on when
 shutting down.
 
-## SAVE and NOSAVE modifiers
+Also note: If Redis receives one of the signals `SIGTERM` and `SIGINT`, the same shutdown sequence is performed.
+See also [Signal Handling](/topics/signals).
 
-It is possible to specify an optional modifier to alter the behavior of the
-command.
+## Modifiers
+
+It is possible to specify optional modifiers to alter the behavior of the command.
 Specifically:
 
-* **SHUTDOWN SAVE** will force a DB saving operation even if no save points are
-  configured.
-* **SHUTDOWN NOSAVE** will prevent a DB saving operation even if one or more
-  save points are configured.
-  (You can think of this variant as an hypothetical **ABORT** command that just
-  stops the server).
+* **SAVE** will force a DB saving operation even if no save points are configured.
+* **NOSAVE** will prevent a DB saving operation even if one or more save points are configured.
+* **NOW** skips waiting for lagging replicas, i.e. it bypasses the first step in the shutdown sequence.
+* **FORCE** ignores any errors that would normally prevent the server from exiting.
```

Comment:
Not clear that such "any errors" can be? Maybe add "details below" or reference the next section?